woocommerce/woocommerce-gateway-stripe
# Stripe settings tab design update

Fixes #2740 

## Changes proposed in this Pull Request:
- Moved the `Debug mode` checkbox  in the `General` section. (I did not move the `New checkout experience` checkbox yet. It's still under the `Advanced settings`. Asked design if I should move that too).
- Implemented the new design for the `Account details` section. (Did not get rid of the webhook information part which is absent in the design as I felt this information is also helpful to a new merchant. Suggest me otherwise, if you think this should be removed)
- Converted the relevant components to styled components instead of using the CSS files.

### Design 
[xVAk98kbRKUByVTrhPc98g-fi-641_70499](https://href.li/?https://www.figma.com/file/xVAk98kbRKUByVTrhPc98g/?node-id=641-70499)

## Testing instructions
- Go to **WooCommerce > Settings > Payments > Stripe** and then the `Settings` tab.
- The `Debug mode` checkbox should be in the `General` section. Make sure enabling/disabling the debug mode works as before.

![debug mode](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/5f1720c4-97a4-4d4a-9f6e-86f9f870aae5)

- The account details section should follow the new design
![account details section](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/0cefb743-86a8-4a67-824d-57bd99b7933b)
